verifier/src/x509.rs

```diff
@@ -19,9 +12,31 @@ use mbedtls::{
     x509::{Certificate, Profile},
 };
 
+pub type Result<T> = core::result::Result<T, Error>;
+
+/// Error type for decoding and verifying certificates.
+#[derive(Debug, displaydoc::Display, PartialEq, Eq)]
+pub enum Error {
+    /// An error occurred working with MbedTls
```

Comment:
Should this render the mbedtls error? (e.g. `/// An error occurred working with MbedTls: {0}`)

Reply:
Good catch, thanks!